test/e2e/framework/client.go

```diff
@@ -313,6 +313,42 @@ func GetFirstValueFromPromQuery(body []byte) (int, error) {
 	return v, nil
 }
 
+// GetFirstValueFromPromQuery takes a query api response body and returns the
+// value of the first timeseries. If body contains multiple timeseries
+// GetFirstValueFromPromQuery errors.
+func GetFirstFloatValueFromPromQuery(body []byte) (float64, error) {
```

Comment:
there's room to factorize code between GetFirstFloatValueFromPromQuery() and GetFirstValueFromPromQuery().

Reply:
Their common logics are going to extract into private function `getFirstRawValueFromPromQuery(body []byte) (*gabs.Container, error)`